crates/pop-cli/Cargo.toml

```diff
@@ -63,8 +63,9 @@ subxt-signer.workspace = true
 default = ["chain", "telemetry", "wasm-contracts"]
 contract = ["wasm-contracts"]
 contracts = ["polkavm-contracts"]
-experimental = ["hashing"]
```

Comment:
It needs rebase here after merging https://github.com/r0gue-io/pop-cli/commits/main/